### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Reviewing structure.

### CODE REVIEW

**Review by aaronrothschild — APPROVED** [review]

File: site/content/integrate/apps/_index.md [hunk 1]
```diff
@@ -13,15 +13,26 @@ Apps are lightweight, interactive add-ons to Mattermost which can use any HTTP-c
 - Receive webhooks from Mattermost, and third-parties, and use the Mattermost REST APIs to post messages, etc.
 - Work on both Mobile and Desktop clients so developers can focus on the functionality of their apps.
 - Be deployed using our serverless hosting infrastructure keeping data secure and supporting scalability by being stateless.
+- Create dynamic fields.
+- Include code as an on-demand function.
```
Comment (hanzei) [inline site/content/integrate/apps/_index.md:17]: What does an that mean?

File: site/content/integrate/apps/_index.md [hunk 2]
```diff
@@ -13,15 +13,26 @@ Apps are lightweight, interactive add-ons to Mattermost which can use any HTTP-c
 - Receive webhooks from Mattermost, and third-parties, and use the Mattermost REST APIs to post messages, etc.
 - Work on both Mobile and Desktop clients so developers can focus on the functionality of their apps.
 - Be deployed using our serverless hosting infrastructure keeping data secure and supporting scalability by being stateless.
+- Create dynamic fields.
```
Comment (hanzei) [inline site/content/integrate/apps/_index.md:16]: What does `fields` refer to? The fields in a form?

File: site/content/integrate/apps/_index.md [hunk 3]
```diff
@@ -13,15 +13,26 @@ Apps are lightweight, interactive add-ons to Mattermost which can use any HTTP-c
...
+
 - Add buttons to channel headers, menus
+
+![image](app-channel-header.png)
+
 - Interactive modals
-- Dynamic fields
-- Code as an on-demand function
+
+![image](app-action.png)
```
Comment (hanzei) [inline site/content/integrate/apps/_index.md:35]: That not a modal :wink:

File: site/content/integrate/apps/system-admin-guide/_index.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+title: "System Admin Guide"
+heading: "System Admin Guide"
+description: "System Admin Guide for Mattermost Apps."
+weight: 90
```
Comment (hanzei) [inline site/content/integrate/apps/system-admin-guide/_index.md:5]: Is it worth adding this right now?

**Review by hanzei — CHANGES_REQUESTED** [review]
a few thoughts below

File: site/content/integrate/apps/system-admin-guide/_index.md (hunk 4 above)
Comment (justinegeffen) [reply site/content/integrate/apps/system-admin-guide/_index.md:5]: No, it's definitely not. I struggled with figuring out the structure so I'll remove it for now.

File: site/content/integrate/apps/_index.md (hunk 3 above)
Comment (justinegeffen) [reply site/content/integrate/apps/_index.md:32]: Yikes. Oops!

File: site/content/integrate/apps/_index.md [hunk 5]
```diff
@@ -13,15 +13,23 @@ Apps are lightweight, interactive add-ons to Mattermost which can use any HTTP-c
...
 
 - Create slash commands
+
+![image](app-slash-command.png)
+
 - Post messages to channels
+
+![image](app-bot.png)
+
 - Add buttons to channel headers, menus
```
Comment (hanzei) [inline site/content/integrate/apps/_index.md:28]:
```suggestion
- Add buttons to channel headers and post menus
```

**Review by hanzei — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**